packages/main/src/Input.js

```diff
@@ -1178,9 +1178,11 @@ class Input extends UI5Element {
 			return;
 		}
 
+		const innerInput = this.getInputDOMRefSync();
```

Comment:
why is this needed? Does it have to do something with the tokenizer and the change?

Reply:
No, I will remove all changes in this file, they are included by mistake from one of my other fixes